api/controller.py

```diff
@@ -952,10 +952,16 @@ def series(self, series_name):
 
         lane = SeriesLane(self._db, series_name)
         annotator = self.manager.annotator(lane)
+        facets = load_facets_from_request()
+        if isinstance(facets, ProblemDetail):
+            return facets
+        pagination = load_pagination_from_request()
+        if isinstance(pagination, ProblemDetail):
+            return pagination
```

Comment:
Also the reason we're seeing this is that the contributor and series feeds have links with pagination and facets that don't work - I made an issue for that too https://github.com/NYPL-Simplified/circulation/issues/344